src/views/router_views/email/submitted.njk

```diff
@@ -0,0 +1,15 @@
+{% extends "layouts/default.njk" %}
```

Comment:
This can be taken out and replaced by `update-submitted`

Reply:
done